<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ECG Recording</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ecg_icon.png</normaloff>ecg_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="GraphicsView" name="ECG_plot"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="control_layout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="title_data_label">
         <property name="text">
          <string>Patient's information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Full Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_line"/>
       </item>
       <item>
        <widget class="QLabel" name="age_label">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="age_line"/>
       </item>
       <item>
        <widget class="QLabel" name="sex_label">
         <property name="text">
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sex_box">
         <item>
          <property name="text">
           <string>Select</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Man</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Woman</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="communication_layout">
       <item>
        <widget class="QPushButton" name="connect_button">
         <property name="text">
          <string>Connect Arduino</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toggle_recording_button">
         <property name="text">
          <string>Start Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="filter_layout">
       <item>
        <widget class="QLabel" name="title">
         <property name="text">
          <string>Filters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noise_line_remover">
         <property name="text">
          <string>Line Noise Removal</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="line_frequency_layout">
         <item>
          <widget class="QLabel" name="frequency_descriptor">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>50Hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60Hz</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="passband_filter">
         <property name="text">
          <string>Pass-Band Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
